src/hellogo_fares/read.rs

```diff
@@ -169,7 +169,7 @@ fn get_line_id(fare_frame: &Element, service_frame: &Element) -> Result<String>
                     .map(|id| id == line_ref)
                     .unwrap_or(false)
             })
-            .map(|line| utils::get_value_in_keylist(line, "KV1PlanningLijnNummer"))
+            .map(|line| netex_utils::get_value_in_keylist(line, "KV1PlanningLijnNummer"))
```

Comment:
It's really too strange to have Netex and KV1 mixed up. 
It should better with a generic name (like TransModel or something else).

Reply:
HelloGo Fares is Netex. KV1 is for the Public Transport data but the fares in HelloGo is using Netex format.

Reply:
Okay. My memory is lacking on this Monday morning :laughing: